vscode/src/webview/webview.tsx

```diff
@@ -49,19 +50,78 @@ type CircuitState = {
   props: CircuitProps;
 };
 
+type DocumentationState = {
+  viewType: "documentation";
+  contentToRender: string;
+};
+
 type State =
   | { viewType: "loading" }
   | { viewType: "help" }
   | HistogramState
   | EstimatesState
-  | CircuitState;
+  | CircuitState
+  | DocumentationState;
 const loadingState: State = { viewType: "loading" };
 const helpState: State = { viewType: "help" };
 let state: State = loadingState;
 
+const themeAttribute = "data-vscode-theme-kind";
```

Comment:
Hopefully the comments are explanatory, but this is what watches for theme changes in VS Code and switches the stylesheet as needed.